dev/deps/spark-deps-hadoop-3.2-hive-2.3

```diff
@@ -161,7 +161,7 @@ metrics-json/4.1.1//metrics-json-4.1.1.jar
 metrics-jvm/4.1.1//metrics-jvm-4.1.1.jar
 minlog/1.3.0//minlog-1.3.0.jar
 netty-all/4.1.51.Final//netty-all-4.1.51.Final.jar
-objenesis/2.6//objenesis-2.6.jar
+objenesis/3.1//objenesis-3.1.jar
```

Comment:
Interesting that this is changed by only changing test dependencies. Is it because jmock-junit4 should be test scope?